src/search.cpp

```diff
@@ -152,6 +152,19 @@ Search::Worker::Worker(SharedState&                    sharedState,
     clear();
 }
 
+// Function P interpolates parameter dependent on number of threads given values for one and eight threaded search.
+// For 8 threads exactly v8 is returned but because of rounding for one thread a value near v1 is returned (at most one unit of).
+
+inline int Search::Worker::P(int v1, int v8) const {
+
+    assert(threads.size() > 0);
+
```

Comment:
`msb` has a similar assertion as well. So if the only purpose of this `assert` is to detect cases where `msb` would be called with `0`, it can be removed.